openeo_udp/biopar/biopar.json

```
@@ -73,83 +73,119 @@
             "optional": true,
             "schema": {
                 "type": "string",
-                "enum": ["FAPAR", "LAI", "FCOVER", "CCC", "CWC"]
+                "enum": [
+                    "FAPAR",
+                    "LAI",
+                    "FCOVER",
+                    "CCC",
+                    "CWC"
+                ]
             }
         }
     ],
     "process_graph": {
-        "adddimension1": {
+        "loadcollection1": {
+            "process_id": "load_collection",
             "arguments": {
-                "data": {
-                    "from_node": "reducedimension1"
-                },
-                "label": {
-                    "from_parameter": "biopar_type"
-                },
-                "name": "bands",
-                "type": "bands"
-            },
-            "process_id": "add_dimension",
-            "result": true
+                "bands": [
+                    "B03",
+                    "B04",
+                    "B08",
+                    "sunAzimuthAngles",
+                    "sunZenithAngles",
+                    "viewAzimuthMean",
+                    "viewZenithMean"
+                ],
+                "id": "SENTINEL2_L2A",
+                "spatial_extent": null,
+                "temporal_extent": null
+            }
         },
-        "filterspatial1": {
+        "filtertemporal1": {
+            "process_id": "filter_temporal",
             "arguments": {
                 "data": {
-                    "from_node": "filtertemporal1"
+                    "from_node": "loadcollection1"
                 },
-                "geometries": {
-                    "from_parameter": "geometries"
+                "extent": {
+                    "from_parameter": "temporal_extent"
                 }
-            },
-            "process_id": "filter_spatial"
+            }
         },
-        "filtertemporal1": {
+        "filterspatial1": {
+            "process_id": "filter_spatial",
             "arguments": {
                 "data": {
-                    "from_node": "loadcollection1"
+                    "from_node": "filtertemporal1"
                 },
-                "extent": {
-                    "from_parameter": "temporal_extent"
+                "spatial_extent": {
```

the argument of `filter_spatial` is "geometries", not "spatial_extent"
```suggestion
                "geometries": {
```